packages/glibc_build223.rb

```diff
@@ -335,7 +321,7 @@ def self.install
     end
     # Only save libnsl.so.2, since libnsl.so.1 is provided by perl
     # For this to work, build on a M107 or newer container.
-    FileUtils.cp File.realpath("#{CREW_DEST_LIB_PREFIX}/libnsl.so.1"), "#{CREW_DEST_LIB_PREFIX}/libnsl.so.2"
+    FileUtils.ln_sf File.realpath("#{CREW_DEST_LIB_PREFIX}/libnsl.so.1"), "#{CREW_DEST_LIB_PREFIX}/libnsl.so.2"
```

Comment:
```
 FileUtils.ln_sf File.realpath("#{CREW_LIB_PREFIX}/libnsl.so.1"), "#{CREW_DEST_LIB_PREFIX}/libnsl.so.2
```
?

Reply:
No. We're looking for what the symlink we may have just created in dest_lib_prefix actually points to.

Reply:
This is for symlinking system libraries.

The first argument to FileUtils.ln_sf is the source file, which is actually a system file in /lib{64} or /usr/lib{64}, and we are looking at symlinks to such files that are already in CREW_DEST_LIB_PREFIX, which with `realpath()` will resolve to the /lib{64} or /usr/lib{64} paths.

The second argument is for where we are making the symlink, which is in CREW_DEST_LIB_PREFIX. We want that there so archiving saves the new symlink.